set.mm

```diff
@@ -630918,37 +630918,41 @@ standardize vectors to a length (norm) of one, but such definitions require
   $}
 
   ${
-    cu3addi.1 $e |- A e. CC $.
-    cu3addi.2 $e |- B e. CC $.
-    cu3addi.3 $e |- C e. CC $.
-    $( Cube of sum of three numbers.  (Contributed by Igor Ieskov,
-       14-Dec-2023.) $)
-    cu3addi $p |- ( ( ( A + B ) + C ) ^ 3 ) =
-    ( ( ( ( ( A ^ 3 ) + ( 3 x. ( ( A ^ 2 ) x. B ) ) )
-    + ( ( 3 x. ( A x. ( B ^ 2 ) ) ) + ( B ^ 3 ) ) )
-    + ( ( ( 3 x. ( ( A ^ 2 ) x. C ) )
-    + ( ( ( 3 x. 2 ) x. ( A x. B ) ) x. C ) )
-    + ( 3 x. ( ( B ^ 2 ) x. C ) ) ) )
-    + ( ( ( 3 x. ( A x. ( C ^ 2 ) ) )
-    + ( 3 x. ( B x. ( C ^ 2 ) ) ) ) + ( C ^ 3 ) ) )
-    $=
-      ( caddc co c3 cexp c2 cmul wceq wtru cc wcel a1i cu3addd mptru ) ABGZHCTH
-      IZJZHAUAUBHUAAKZUBHZBLZHUEHTHUAABUCUBHZUEHUEHBUAUBHTHTHUAUDCUEHUEHUAUCUEH
-      ABUEHUEHCUEHTHUAUFCUEHUEHTHTHUAACUCUBHZUEHUEHUABUGUEHUEHTHCUAUBHTHTHMNZAB
-      CAOZPUHDQBUIPUHEQCUIPUHFQRS $.
+    sqnegd.1 $e |- ( ph -> A e. CC ) $.
+    $( The square of the negative of a number.  (Contributed by Igor Ieskov,
+       21-Jan-2024.) $)
+    sqnegd $p |- ( ph -> ( -u A ^ 2 ) = ( A ^ 2 ) ) $=
+      ( cc wcel cneg c2 cexp co wceq sqneg syl ) ABDEBFGZHZIBMNIJCBKL $.
+  $}
+
+  ${
+    anpnaneq0d.1 $e |- ( ph -> A e. RR ) $.
+    anpnaneq0d.2 $e |- ( ph -> N e. NN0 ) $.
+    anpnaneq0d.3 $e |- ( ph -> -. 2 || N ) $.
+    $( The sum of a real number to the power of N and the negative of the
+       number to the power of N equals zero if N is a nonnegative even integer.
```

Comment:
Should be "nonnegative **odd** integer".

Reply:
Fixed. Thanks.